packages/eslint-config-react-app/index.js

```diff
@@ -26,13 +26,21 @@ module.exports = {
   plugins: ['import', 'flowtype', 'jsx-a11y', 'react'],
 
   env: {
-    browser: true,
     commonjs: true,
     es6: true,
     jest: true,
     node: true,
   },
 
+  globals: {
+    document: true,
```

Comment:
Let's change this to the list of globals defined by Standard: `document`, `console`, and `navigator`.